jsref/Makefile

```diff
@@ -217,8 +216,9 @@ Translate_syntax.cmo: Translate_syntax.ml Translate_syntax.mli Translate_syntax.
 	ocamlc -c $(REPLACE_STDLIB) $(PPX_FLAG) $(OCAMLCFLAGS) $<
 
 clean_mljsref:
-	sh -c "rm -f $(ML_FRONTEND_SRC)/*.mli"
-	sh -c "rm -f {,$(ML_FRONTEND_SRC)/}*.{cmo,byte}"
```

Comment:
I think the original clean command here was broken:  `$(ML_FRONTEND_SRC)` isn't defined anywhere in the Makefile (I think it was left in from a previous version).

{,$(ML_...)} also expanded out the empty string to give `rm -f *.cmo` etc.

The `.mli` line can be removed, and the other two altered to just remove the files in the current directory.

Thanks for fixing this.

Reply:
I have seen that during the fix and i was wondering if i had to remove it, il'll remove it.